index.bs

```diff
@@ -3206,8 +3217,10 @@ The following abstract operations support the implementation of the
  1. Let |firstDescriptor| be |controller|.[=ReadableByteStreamController/[[pendingPullIntos]]=][0].
  1. If |firstDescriptor|'s [=pull-into descriptor/byte offset=] + |firstDescriptor|' [=pull-into
     descriptor/bytes filled=] is not |view|.\[[ByteOffset]], throw a {{RangeError}} exception.
- 1. If |firstDescriptor|'s [=pull-into descriptor/byte length=] is not |view|.\[[ByteLength]], throw
-    a {{RangeError}} exception.
+ 1. If |firstDescriptor|'s [=pull-into descriptor/buffer byte length=] is not
+    |view|.\[[ViewedArrayBuffer]].\[[ByteLength]], throw a {{RangeError}} exception.
+ 1. If |firstDescriptor|'s [=pull-into descriptor/bytes filled=] + |view|.\[[ByteLength]] >
+    |firstDescriptor|'s [=pull-into descriptor/byte length=], throw a {{RangeError}} exception.
```

Comment:
This repeats the check in the first step of [ReadableByteStreamControllerRespondInReadableState](https://streams.spec.whatwg.org/#readable-byte-stream-controller-respond-in-readable-state). But [as I mentioned](https://github.com/whatwg/streams/pull/1123#issuecomment-821727018), we throw this error too late... 😕